presto-native-execution/presto_cpp/main/common/Configs.cpp

```diff
@@ -189,6 +189,7 @@ SystemConfig::SystemConfig() {
           NUM_PROP(kAsyncCacheMaxSsdWriteRatio, 0.7),
           NUM_PROP(kAsyncCacheSsdSavableRatio, 0.125),
           NUM_PROP(kAsyncCacheMinSsdSavableBytes, 1 << 24 /*16MB*/),
+          STR_PROP(kAsyncCacheFullPersistenceInterval, "0s"),
```

Comment:
@zacw7 : Is it possible to use a negative value here instead ? 0 gives the impression that it always runs (though the code doesn't run it)

Reply:
To avoid confusion I've updated the comment, thanks for pointing that out!

I would prefer keep 0 as the default value here as it seems more aligned with the convention of similar configs (ex. https://github.com/prestodb/presto/pull/22764). If more folks think it gives the wrong impression I'm open to have it changed to a negative value? WDYT @tanjialiang